proto/http_types.proto

```diff
@@ -33,4 +33,13 @@ message Scheme {
     Registered registered = 1;
     string unregistered = 2;
   }
+}
+
+message Headers {
+  message Header {
+    string name = 1;
+    bytes value = 2;
```

Comment:
curious if you've already given any thought as to how to render binary headers.

Reply:
i think the current plan is to largely let grpc-go's json encoder's reckon with it

Reply:
linkerd/linkerd2#3411 JSON encodes tap events. `json` does not attempt to decode `[]byte` values. I think if we wanted to decode these values into possibly valid UTF8, it may need to happen [here](https://github.com/linkerd/linkerd2/pull/3411/files#diff-d8e7251a1dbf9f4433c25cfa8f933172R363), where we set the header value in the public `TapEvent` value field.

If it is not valid UTF8, we could keep it base64 encoded, print something like `-bin`, or something else.

Reply:
Answered [here](https://github.com/linkerd/linkerd2/pull/3411#issuecomment-530098126). This change should continue treating header values as `bytes` and the control plane will attempt to decode the values for displaying to the user.